src/unistd.rs

```diff
@@ -191,10 +191,21 @@ impl ForkResult {
 /// Continuing execution in parent process, new child has pid: 1234
 /// I'm a new child process
 /// ```
+///
+/// # Safety
+///
+/// In a multithreaded program, only [async-signal-safe] functions may be called
```

Comment:
Your comments talk about `libc::exit` and `libc::pause` being safe to use here. Let's explicitly list those here. We should also state that `std::process:exit` is not safe.